client/templates/hangout/hangout-item.js

```diff
@@ -47,17 +47,18 @@ Template.hangoutItem.helpers({
   },
   getDate: function(hangout) {
     var tz = TimezonePicker.detectedZone();
+    var startEndDiffDays = (hangout.end - hangout.start)/(1000*60*60*24)
+    var startTime = moment(hangout.start).tz(tz).format('ddd MMMM Do YYYY, h:mm a z')
+    // If endtime is in 24 hours (95% of 24 hours), then show "To Be Announced". Else show the end time
+    var endTime = startEndDiffDays < 0.95 ? moment(hangout.end).tz(tz).format('MMMM Do h:mm a z'): 'To Be Announced '
+    var joinedUsers = hangout.users.length + ' joined'
```

Comment:
I can move the " joined" into the template.

In terms of the naming convention of the getDate method, because I am not sure of all the places that it is used. I am new to meteor and javaScript and even though seems like there is a 1 to 1 ratio between the templates and the js files, I do not know if that is true. If this is true, then I have no problem changing the name of that method.

I am also not sure what to call it (given that I changed the name). If you have suggestions, feel free to pass them along.